### PR TITLE
feat: [#187611792] enable public accountant license

### PR DESCRIPTION
**Pull request by AnJuHyppolite**
## Description

Given I am a Poppy or Dakota and I am in the Accounting industry
When accessing the Public Accountant license task in the roadmap
Then I can check the status of my license with Consumer Affairs Dynamics system

### Ticket

[#187611792](https://www.pivotaltracker.com/story/show/187611792)

### Steps to Test

1. Access the site as a Poppy or Dakota in the Accounting Industry. 
2. Click the `Apply for a Certified Public Accountant License` in the roadmap. 
3. Enter the contact information for an expired business. 
4. Go back to the dashboard.
5. Click the Deadlines and Opportunities card and complete the information in the modal.
6. Click the Funding Opportunities card.

The quick action tiles will appear on the dashboard.

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/test/pages/taskUrlSlug.test.tsx [hunk 1]
```diff
@@ -223,63 +223,23 @@ describe("task page", () => {
...
-
-  it("loads License task screen for hvac-license", () => {
-    renderPage(generateTask({ id: "hvac-license" }), generateBusiness({ licenseData: undefined }));
-    expect(screen.getByTestId("cta-secondary")).toBeInTheDocument();
-  });
-
-  it("loads License task screen for appraiser-license", () => {
-    renderPage(generateTask({ id: "appraiser-license" }), generateBusiness({ licenseData: undefined }));
-    expect(screen.getByTestId("cta-secondary")).toBeInTheDocument();
+  describe("License task", () => {
```
Comment (adidner) [inline web/test/pages/taskUrlSlug.test.tsx:226]: Thats a nice change

File: web/src/components/TaskPageSwitchComponent.tsx [hunk 2]
```diff
@@ -62,6 +62,9 @@ export const TaskPageSwitchComponent = ({
     "search-business-name": (
       <BusinessFormation task={getTaskFromRoadmap(roadmap, task.id)} displayContent={displayContent} />
     ),
+    "public-accountant-license": (
+      <LicenseTask task={task} CMS_ONLY_disable_overlay={CMS_ONLY_disable_overlay} />
+    ),
```
Comment (jphechter) [inline web/src/components/TaskPageSwitchComponent.tsx:67]: [dust] I might have put this up with the other `LicenseTask`s just so things are grouped together.

Comment (AnJuHyppolite) [reply web/src/components/TaskPageSwitchComponent.tsx:67]: I started to do this and then changed course. I should have gone with my first mind! I'll change it. Thanks, JP.

File: content/src/roadmaps/industries/certified-public-accountant.json [hunk 3]
```diff
@@ -51,6 +51,12 @@
       "licenseTask": "register-accounting-firm",
       "required": false,
       "weight": 9
+    },
+    {
+      "step": 4,
+      "licenseTask": "public-accountant-license",
+      "required": false,
+      "weight": 10
```
Comment (jphechter) [inline content/src/roadmaps/industries/certified-public-accountant.json:59]: [boulder] This doesn't need to be here. The question is a part of the CPA add-on which is in `cpa.json`. There's an onboarding question `isCpaRequiredApplicable` that controls whether or not the add-on is applied. Adding to the roadmap will result in this task being added twice for the accounting industry.

Comment (adidner) [reply content/src/roadmaps/industries/certified-public-accountant.json:59]: I was going to say the same thing

**Pull request merged**